Source/Core/Core/RmObjEngine.cpp

```diff
@@ -0,0 +1,173 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2
+// Refer to the license.txt file included.
+
+
+// RmObjEngine
+// Supports the removal of objects/effects from the rendering loop
+
+#include "Core/ConfigManager.h"
+#include "Core/Core.h"
+#include "Core/RmObjEngine.h"
+
+namespace RmObjEngine
```

Comment:
This namespace name could be more fully-formed. `ObjectRemovalEngine` is much more clear at a glance.